File: Fundation/week-02/day-03/src/Variables/Exercises/Cuboid.java
public class Cuboid {
  public static void main(String[] args) {
    // Write a program that stores 3 sides of a cuboid as variables (doubles)
    double a = 20;
    double b = 45;
    double c = 59;

    // The program should write the surface area and volume of the cuboid like:
    //
    // Surface Area: 600
    // Volume: 1000

    double surface = 2 * ((a * b) + (b * c) + (a * c));
    System.out.println("Surface Area: " + surface);

    double volume = a * b * c;
    System.out.println("Volume: " + volume);
  }
}

File: Project/week-21/day-02/Basic JS/Variables/cuboid.ts
'use strict';

// Write a program that stores 3 sides of a cuboid as variables (floats)
// The program should write the surface area and volume of the cuboid like:
//
// Surface Area: 600
// Volume: 1000

let sideA: number = 2;
let sideB: number = 3;
let sideC: number = 4;

console.log('Surface Area: ' + 2*(sideA*sideB + sideA*sideC + sideB*sideC));
console.log('Volume: ' + sideA*sideB*sideC);